packages/frontend/src/main/pages/admin/ServerSettings.vue

```diff
@@ -106,7 +107,7 @@ export default {
           }
         `,
         variables: {
-          info: this.serverModifications
+          info: pick(this.serverModifications, Object.keys(this.serverDetails))
```

Comment:
Fixed a bug that had broken the server settings form so that you couldn't submit it. It sends too many fields to the backend essentially.